Vagrantfile

```diff
@@ -2,7 +2,7 @@
 # vi: set ft=ruby :
 
 Vagrant.configure('2') do |config|
-    config.vm.box = 'ubuntu/bionic64'
+    config.vm.box = 'ubuntu/mantic64'
```

Comment:
Why not some LTS version? It seems that `mantic` is `23.10`. `bionic` is an LTS version.

Reply:
Newest LTS version seems to be `noble`

Reply:
`noble` is not available as vagrant box https://app.vagrantup.com/ubuntu